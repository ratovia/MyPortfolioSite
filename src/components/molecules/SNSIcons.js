import React from "react"
import styled from "styled-components"
import { CircleImageIcon } from "../atoms/CircleImageIcon"
import ratoviaIcon from "../../../static/ratovia_icon.jpg"

const SNSArea = styled.div`
  display: flex;
  justify-content: space-between;
  width: 280px;
`

export const SNSIcons = () => {
  return (
    <SNSArea>
      <CircleImageIcon imageUrl={ratoviaIcon} size="small"></CircleImageIcon>
      <CircleImageIcon imageUrl={ratoviaIcon} size="small"></CircleImageIcon>
      <CircleImageIcon imageUrl={ratoviaIcon} size="small"></CircleImageIcon>
      <CircleImageIcon imageUrl={ratoviaIcon} size="small"></CircleImageIcon>
    </SNSArea>
  )
}
