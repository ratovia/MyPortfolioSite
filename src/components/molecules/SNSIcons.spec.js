import React from "react"
import renderer from "react-test-renderer"
import { SNSIcons } from "./SNSIcons"

describe("SNSIcons", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<SNSIcons />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
