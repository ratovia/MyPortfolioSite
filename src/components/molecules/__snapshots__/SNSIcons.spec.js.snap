// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SNSIcons renders correctly 1`] = `
<div
  className="sc-gsDJrp dtcqJz"
>
  <img
    alt="アイコン"
    className="sc-bdvvaa hVPPTo"
    height="48px"
    src="test-file-stub"
    width="48px"
  />
  <img
    alt="アイコン"
    className="sc-bdvvaa hVPPTo"
    height="48px"
    src="test-file-stub"
    width="48px"
  />
  <img
    alt="アイコン"
    className="sc-bdvvaa hVPPTo"
    height="48px"
    src="test-file-stub"
    width="48px"
  />
  <img
    alt="アイコン"
    className="sc-bdvvaa hVPPTo"
    height="48px"
    src="test-file-stub"
    width="48px"
  />
</div>
`;
